Manito/Manito/Presentation/Scene/SplashScene/ViewController/SplashViewController.swift

```diff
@@ -76,7 +76,7 @@ final class SplashViewController: UIViewController {
 // MARK: - Helper
 extension SplashViewController {
     private func presentLoginViewConroller() {
-        let viewController = LoginViewController()
+        let viewController = LoginViewController(viewModel: LoginViewModel(loginService: LoginService(repository: LoginRepositoryImpl())))
```

Comment:
p1;

이 부분은 가독성을 위해 
```suggestion
        let repository = LoginRepositoryImpl()
        let service = LoginService(repository: repository)
        let viewModel = LoginViewModel(loginService: service)
        let viewController = LoginViewController(viewModel: viewModel)
```

이런식으로 하기로 했습니다! 수정부탁드려요!

Reply:
[0f24003](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/580/commits/0f240037fd9abb67cad80feeea6da0152cdb538c)
수정했습니다.